### PR TITLE
Optimising microservices part two

### PR DESCRIPTION
**Pull request by jmunro89**
Part 2 added

### CODE REVIEW

**Comment by adamhepton** [comment]
Looks good to me, I really like how the article is in small, bitesize sections which form part of a larger whole.  Await pendants for further grammatical comment...

**Comment by muggahtee** [comment]
>  Await pendants for further grammatical comment...

pedants*

Or did I just walk into your trap?

**Comment by adamhepton** [comment]
Yes.  Yes, you did.

File: _posts/2018-11-08-optimising-microservices-part-2.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+One of the good things about our new VPC is that it allows us to get the benefits of another AWS caching service, Amazon DynamoDB Accelerator (DAX). DAX puts an in-memory cache in front of your DynamoDB that delivers up to a 10x performance improvement:
+[About DynamoDB Accelerator](https://aws.amazon.com/dynamodb/dax/)
+
+We couldn't use this before because it requires anything that needs access to the DB to be sat within a VPC, but now we have one we might as well get those added benefits too. Anything to shave off those pesky milliseconds!
+
+## The results are in...
+
+After adding our improved caching, we reran our test and our latest 95<sup>th</sup> percentile latency is… **92ms!** A 70% decrease on our baseline of 320ms.
+
+AWS X-Ray provides latency histogram data that can be accessed via the AWS Cli. Here's a before and after chart of our internal system latency:
```
Comment (cmbuckley) [inline _posts/2018-11-08-optimising-microservices-part-2.md]:
```suggestion
AWS X-Ray provides latency histogram data that can be accessed via the AWS CLI. Here's a before and after chart of our internal system latency:
```

**Pull request merged**